AsyncDisplayKit/TextKit/ASTextKitRenderer.mm

```diff
@@ -82,7 +82,20 @@ - (void)_calculateSize
   CGRect constrainedRect = {CGPointZero, _constrainedSize};
   __block CGRect boundingRect;
   [_context performBlockWithLockedTextKitComponents:^(NSLayoutManager *layoutManager, NSTextStorage *textStorage, NSTextContainer *textContainer) {
+    NSParagraphStyle *paragraphStyle = [textStorage attributesAtIndex:[layoutManager characterIndexForGlyphAtIndex:0]
```

Comment:
What if the paragraph style changes at some point through the text?  Just not supported?